test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/TestPrivilegedMode.java

```diff
@@ -35,6 +35,7 @@
  * @summary Test that IR framework successfully adds test class to boot classpath in order to run in privileged mode.
  * @modules java.base/jdk.internal.vm.annotation
  * @library /test/lib /
+ * @build jdk.test.lib.Platform
```

Comment:
Is that really required or is it enough to set `TEST_CLASS_PATH`?

Reply:
Yes, it's enough to set `TEST_CLASS_PATH`. But the jtreg [doc](https://openjdk.org/jtreg/tag-spec.html) suggest that `appropriate @build directives to ensure that the classes will be compiled`.

Reply:
Okay, but then I suggest to remove `@build` because the IR framework is already special in that regard as it aims for simplicity: The IR test itself should not need to worry about the IR framework internal classes that need to be built. For example, we already silently build and install the `WhiteBox` class without the need to specify it with `@build` in the IR test.

Reply:
Okey, the `@build jdk.test.lib.Platform` has been deleted.